src/main/java/Duke.java

```diff
@@ -1,10 +1,123 @@
+                    }
+                    break;
+                case "todo":
+                    try {
+                        String rest = input.substring(command.length());
+                        if (rest.equals("")) {
+                            throw new DukeException("____________________________________________________________\n" +
+                                    " ☹ OOPS!!! The description of a todo cannot be empty.\n" +
+                                    "____________________________________________________________");
+                        }
+                        String[] arr = rest.split("/");
+                        Todo item = new Todo(arr[0]);
+                        tasks.add(item);
+                        System.out.println("____________________________________________________________\n Got it. I've added this task:\n"
+                                + "   " + item + "\n Now you have " + tasks.size() + " tasks in the list."
+                                + "\n____________________________________________________________");
+                    } catch (DukeException e) {
+                        System.out.println(e.getMessage());
+                    }
+                    break;
+                case "event":
+                    try {
+                        String rest = input.substring(command.length());
+                        if (rest.equals("")) {
+                            throw new DukeException("____________________________________________________________\n" +
+                                    " ☹ OOPS!!! The description or time/date of an event cannot be empty.\n" +
+                                    "____________________________________________________________");
+                        }
+                        String[] arr = rest.split("/");
+                        Event item = new Event(arr[0], arr[1].substring(5), arr[2].substring(3));
+                        tasks.add(item);
+                        System.out.println("____________________________________________________________\n Got it. I've added this task:\n"
+                                + "   " + item + "\n Now you have " + tasks.size() + " tasks in the list."
+                                + "\n____________________________________________________________");
+                    } catch (DukeException e) {
+                        System.out.println(e.getMessage());
+                    }
+                    break;
+                case "mark": {
+                    int num = Integer.parseInt(input.split(" ")[1]);
+                    tasks.get(num - 1).markDone();
+                    System.out.println("____________________________________________________________\n" +
+                            "Nice! I've marked this task as done:\n" +
+                            "   " + tasks.get(num - 1).toString() + "\n" +
+                            "____________________________________________________________");
```

Comment:
I think you could make the line breaks private fields so that you don't need to type so many times